Supported-Annotations.md

```diff
@@ -30,4 +30,13 @@ public class NullnessChecker {
 
 For now, the `@Contract` annotations are trusted, not checked.  NullAway will warn if it sees a call to a method with a `@Contract` annotation it recognizes as invalid. 
 
-Not all possible clauses of `@Contract` annotations are fully parsed or supported by NullAway (e.g. `@Contract("null, false -> fail; !null, true -> fail")` will be ignored, since NullAway cannot generally reason about the runtime truth value of the second argument).
\ No newline at end of file
+Not all possible clauses of `@Contract` annotations are fully parsed or supported by NullAway (e.g. `@Contract("null, false -> fail; !null, true -> fail")` will be ignored, since NullAway cannot generally reason about the runtime truth value of the second argument).
+
+### Field Contracts (postcondition and precondition)
```

Comment:
This requires more explanation on what the annotations are for and how to use them, not just their allowed parameters.